src/modules/app/route-configs/LocalNewsDetailsRouteConfig.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import type { AllPayloadsType } from './RouteConfig'
+import { RouteConfig } from './RouteConfig'
+import type { Route } from 'redux-first-router'
+import {
+  createCitiesEndpoint,
+  createEventsEndpoint,
+  createLocalNewsElementEndpoint,
+  createLanguagesEndpoint,
+  LocalNewsModel,
+  Payload
+} from '@integreat-app/integreat-api-client'
+import fetchData from '../fetchData'
+import { cmsApiBaseUrl } from '../constants/urls'
+
+type LocalNewsDetailsType = {| city: string, language: string |}
+type RequiredPayloadsType = {| localNewsDetails: Payload<LocalNewsModel> |}
+
+export const LOCAL_NEWS_DETAILS_ROUTE = 'LOCAL_NEWS_DETAILS'
+
+const localNewsDetailsRoute: Route = {
+  path: '/:city/:language/news/local/:id',
+  thunk: async (dispatch, getState) => {
+    const state = getState()
+    const { city, language, id } = state.location.payload
+
+    await Promise.all([
+      fetchData(createLocalNewsElementEndpoint(cmsApiBaseUrl), dispatch, state.newsElement, { city, language, id }),
+      fetchData(createCitiesEndpoint(cmsApiBaseUrl), dispatch, state.cities),
+      fetchData(createEventsEndpoint(cmsApiBaseUrl), dispatch, state.events, { city, language }),
+      fetchData(createLanguagesEndpoint(cmsApiBaseUrl), dispatch, state.languages, { city, language })
+    ])
+  }
+}
+
+class LocalNewsDetailsRouteConfig implements RouteConfig<LocalNewsDetailsType, RequiredPayloadsType> {
+  name = LOCAL_NEWS_DETAILS_ROUTE
+  route = localNewsDetailsRoute
+  isLocationLayoutRoute = true
+  requiresHeader = true
+  requiresFooter = true
+
+  getLanguageChangePath = ({ location, payloads, language }) => null
```

Comment:
why we return null here

Reply:
Language shouldn't change for localNewsDetails route so we return `null` here. same as other routes